.gitattributes

```diff
@@ -0,0 +1 @@
+*.cs eol=lf
```

Comment:
we have to normalize the line endings for this to work. So if you are not publishing artefacts from a ci system, you should probably reclone the repo so your line endings are consistent.

Reply:
well...re-clone once this is merged